templates/payment_success.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Registration Successful</title>
+    <link rel='stylesheet prefetch'
+        href='https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0/css/bootstrap.css'>
+
+</head>
+
+<body>
+
+    <div class="container">
+        <h2>You are registered for the DESC August 2022 collaboration meeting!</h2>
+        <br>
+
+        <div class="alert alert-info" role="alert">
+            You should have received by email a confirmation of your payment from <i>maureenlowery@uchicago.edu</i>. You
+            might notice that <b>the receipt will mention CMB Pol Workshop but this is normal</b>. If you have any
+            questions
+            regarding payment, please reach out to the LOC.
```

Comment:
Can we be more explicit about how to reach out to LOC?  Email? Slack channel?

Reply:
added link to hotline channel, confluence is down so can't find link to LOC